.zuul.yaml

```diff
@@ -137,11 +145,13 @@
               - roles/telemetry_autoscaling/.*
               - .zuul.yaml
               - ci/vars-functional-test.yml
-              - ci/run_autoscaling_osp18.yml
+              - ci/run_cloudops_tests_osp18.yml
               - ci/vars-use-master-containers.yml
               - ci/use-master-containers.yml
               - ci/patch-openstack-version.yaml
               - ci/report_result.yml
+              - roles/telemetry_verify_metrics/.*
+              - ci/vars-metric-verification-test.yml
```

Comment:
By adding the extra task in run_cloudops_tests_osp18.yml, you have resolved the differences between the autoscaling and the metrics verification jobs, so you can remove the vars-metric-verification-test.yml from the repo.